pyccel/codegen/python_wrapper.py

```diff
@@ -229,19 +238,18 @@ def create_shared_library(codegen,
         # ...
 
         # Create MOD.so shared library
-        if language == 'fortran':
```

Comment:
Why is this line removed?

Reply:
because we are already in that case if you look here https://github.com/pyccel/pyccel/blob/c0a393a41ca33ab2d4418da348f9d2b9a7220e63/pyccel/codegen/python_wrapper.py#L223